### ISSUE TITLE
New Map Testing

### ISSUE THREAD

**Issue by GhostEcho** [BadDeathclaw/TG-Claw#933]
Report issues with the new map here!

**Comment by MrBobCool** [comment]
1. You already know, radios don't work.
2. Tables called "Wild Plant" in middle of vault. Don't think that should be.. http://prntscr.com/nv85ar
3. Vault stairs don't work, they're called "floor". When you click on them, nothing happens.
4. The Vault library has a lot of bookshelves. Most of the books are placeholder books which have nothing on them. Should add some books.
Yes, all of these are vault things.

**Comment by GhostEcho** [comment]
[Priority]: # (Is it game breaking?) I SPAWNED IN VAULT AS DEN DOCTOR VERY BAD!

Uh?

**Comment by MrBobCool** [comment]
Most sprites for clothing laying around, "Worn Rags, Gentlemens Suit, Ect" has no sprite when equipped.

**Comment by pireamaineach** [comment]
Oh I was lied to then.

**Comment by pireamaineach** [comment]
He said he fixed that.

**Comment by GhostEcho** [comment]
APCs aren't mapped in correctly for some locations, mainly the NCR base

**Comment by GhostEcho** [comment]
Shuttle has nowhere to land

**Comment by pireamaineach** [comment]
Honestly when are we going to remove shuttles entirely in favour of straight timers.

**Comment by Dr-Oreo** [comment]
You cannot dig desert or dirt with a shovel and attempting to place soil on either will give you the message:
 "Needs to be constructed on floor."

**Comment by ProfessorPopoff** [comment]
For sure, even basic signs point to JUST kebabtown can bring people all the bearing they need. Map is crowded but it's nothing that a couple signs can't fix.

**Comment by TheHellFireo** [comment]
> Shuttle has nowhere to land
 pretty sure shuttle is now useless. Chowder is in (or was ) favour of removing the vault and it wouldn't be lore wise a god damn shuttle coming to pick up wastelander you know

**Comment by TheHellFireo** [comment]
> You cannot dig desert or dirt with a shovel and attempting to place soil on either will give you the message:
> "Needs to be constructed on floor."



> Most sprites for clothing laying around, "Worn Rags, Gentlemens Suit, Ect" has no sprite when equipped.

What a nice out of nowhere sprite problem

**Comment by Cactus2388** [comment]
Dispatcher's uniform sprite is broken and is found somewhere in kebab town I think?
NCR doesn't spawn with a defibrillator like normal and detriments them heavily

**Comment by Cactus2388** [comment]
![onfire](https://user-images.githubusercontent.com/27895840/59007221-1e7bb980-87eb-11e9-9310-3bcf453752c8.PNG)
barrels light on fire like this

**Comment by TheHellFireo** [comment]
> ![onfire](https://user-images.githubusercontent.com/27895840/59007221-1e7bb980-87eb-11e9-9310-3bcf453752c8.PNG)
> barrels light on fire like this

That's normal

**Comment by Cactus2388** [comment]
NCR doesn't have a fucking shower what a bunch of smelly profligates

**Comment by pireamaineach** [comment]
When the map is out of testing I'm making a ranger outpost. I can turn the current ranger spawnroom into showers then.

**Comment by Cactus2388** [comment]
![sinkwall](https://user-images.githubusercontent.com/27895840/59008040-b929c780-87ee-11e9-8abf-b9cca274316a.PNG)
Sinks do not belong on walls

**Comment by Cactus2388** [comment]
![BATTLEBUS](https://user-images.githubusercontent.com/27895840/59009328-c0eb6b00-87f2-11e9-8d52-962910d73173.PNG)
YOU CAN DRAG BUSSES AROUND BECAUSE THEY'RE JUST FUCKING CHILDREN OF THE CLOSET AND ITS A LITERAL FUCKING BATTLE BUS THAT PEOPLE CAN'T MELEE LMAO

**Comment by pireamaineach** [comment]
Actually if it's a northernmost wall than according to ss13 perspective the sink should be on the wall.

**Comment by Cactus2388** [comment]
That's a snap from OUTSIDE the bathroom

**Comment by pireamaineach** [comment]
Ah so what you need to do to fix that is nudge the sink south. The sprite will appear on the wall in the bathroom but not appear at all outside.

**Comment by Cactus2388** [comment]
NCR has no food in their kitchen.
![fridgesblockdoors](https://user-images.githubusercontent.com/27895840/59012050-0fe9ce00-87fc-11e9-9106-b5a82a27c45c.PNG)
Fridges block doors in the NCR kitchen
Power in the NCR base is so unmaintainable that the autolathe and lights hardly work without anyone setting power up
THERE'S NO CHEM MACHINE ON THE MAP SO NCR IS STUCK WITH WHAT LITTLE THEY HAVE

**Comment by Cactus2388** [comment]
ED-13 don't have a death sprite

**Comment by Cactus2388** [comment]
![mapping](https://user-images.githubusercontent.com/27895840/59018413-73c7c300-880b-11e9-920a-79940f4fa829.PNG)
the ruins a little south and East of the NCR base still have a mapping thing seeable in-game

**Comment by Zebraman** [comment]
- For some reason I was unable to build any walls in the Legion's base due to lack of floor. I also couldn't build floors to solve this issue.
- The map cruelly lack of tools, and loot in general. The abscence of lathe in Kebab town or other tool vendor across the map make it even worst.
- Most of the fridges and containers are empty.
- Kebab Town's bar doesn't spawn with booze and soda dispenser, forcing the wannabe bartender to go scavenge the only one available in the abandoned bar near the legion base.
- Kebab town itself feel a bit too enclaved. The location would be ok for something like a Klamat equivalent, but it's too cramped and recluded for the main town of the map
- Some place containing barely any loot are guarded by an army, while there's not a single mob near the ammo bench.
- Lack of funny outfits/items to RP, I still didn't manage to find a single clown mask.
- ID dogtag for Legion are invisible

**Comment by DaKittaX** [comment]
Veteran ranger's 50. cal sniper's ammo is invisible.

**Comment by GoodG1448** [comment]
The desert turf sprites repeats itself a lot, we didn't had this issue in the old map. Also, we could use more tallgrass spawning around just for aesthetics (Same thing goes for some of the areas with mobs in it, it could use some blood decals for flavor)

On other topic, there's a building almost at the east end of the map that has a fucking GMOD ERROR sign in it.
![Test](https://user-images.githubusercontent.com/38330373/59045492-8ddac300-8856-11e9-9216-6b3e51048b96.png)

**Comment by GoodG1448** [comment]
There's also this FUCKHUGE empty area just north east outside of Kebab town by the vertical street. We could use something like a bigger mall to loot food.

![Test2](https://user-images.githubusercontent.com/38330373/59045823-2b35f700-8857-11e9-8998-950c6e03d1bb.png)

**Comment by GoodG1448** [comment]
The BoS bunker is missing light bulbs/tubes in the hallways (I think only bulbs would be more fitting since it fits with the bunket aesthetic).

![Test3](https://user-images.githubusercontent.com/38330373/59046318-2e7db280-8858-11e9-884b-d7ea97e4328c.png)

**Comment by TheHellFireo** [comment]
> There's also this FUCKHUGE empty area just north east outside of Kebab town by the vertical street. We could use something like a bigger mall to loot food.
> 
> ![Test2](https://user-images.githubusercontent.com/38330373/59045823-2b35f700-8857-11e9-8998-950c6e03d1bb.png) BoS can make them with their auto lathe.

**Comment by GoodG1448** [comment]
More big fucking empty space, this one is just south of the NCR base.

![Test4](https://user-images.githubusercontent.com/38330373/59046720-dd21f300-8858-11e9-9323-b7d541bf24e9.png)

**Comment by GoodG1448** [comment]
Also, the map has no day-night cycle so we are stuck in eternal daytime.

**Comment by Kabramen** [comment]
there's this road ground in a wrong spot
![image](https://user-images.githubusercontent.com/46802162/59058989-2d5a7e80-8874-11e9-89cd-1b84c0d20448.png)

**Comment by Jackofashepard** [comment]
People are having constant spikes of 2k ping ever since the new map was implemented, most tiles do not count as "floor" making building a bitch, legion lacks any tiles aside from a welder, no autolathes, no defib in NCR, power in NCR was not properly done at all leading to the base having 0 power 100% of the time, guns sometimes refuse to shoot (might be lag/bug, a lot of people reported it and its not the "your gun is not ready" thing) and finally invis ID's and .50 ammo

**Comment by GhostEcho** [comment]
I can pretty much confirm the lag issue is just server not the map itself

**Comment by Jackofashepard** [comment]
Its pretty coincidental since it only started after the new map was implemented

**Comment by GhostEcho** [comment]
Nah, its been happening to me where I get 2k ping on joining the server for like five minutes, and its been happening to others like this for the past couple of weeks

**Comment by Jackofashepard** [comment]
Let me clarify, it happens the entire round

**Comment by GhostEcho** [comment]
Im sure, but again, its really a serverside issue, not a map issue

**Comment by Cactus2388** [comment]
Even IF NCR sets up power there's usually no LIGHTING because people don't want to waste the power in the PACman this is REALLY bad design and I know that even adding uranium wouldn't stop this because it'll still be a valuable material going to "waste" on lighting. Maybe try making lights independent from power consumption in the NCR base?

**Comment by CptPatriot** [comment]
i havent read what anyone else posted, sorry for potential reposts

NCR:
-Showers.
-Either give them a bunch of uranium or actually have some spawn on the map, or just get rid of the stupid generator entirely. It's good to have the idea of maintenance for a base but not on such a scale where it only serves as a massive, otherwise easily resolved, inconvenience. 
-More rooms? There's just mass stretches of emptiness in the NCR base. Add a ranger room preferabley with their belts and binos and stuff, I did like the idea of removing their outpost though. 
-Need defib at round start.
-More than four(?) stimpaks at round start. 

Legion:
-Brass boxes. NCR gets a lathe. They should get a handful of tools as they aways have.
-Oxygen tanks. I'd rather the powerfist just get nerfed than only be able to use it for a few hits.
-At least a couple handheld radios, otherwise legion is pretty crippled right off the bat
-Garden/Kitchen connection is cool but they need to be extended back further.
-Centurion's 'quarters' is pretty shit aesthetically speaking. Literally gets a slave bed and a fridge.
-Actual slave pens and not one room behind a gate that just leads into the garden which will never be locked. Super easy for slaves to run straight to the armory if someone stops looking for .2 seconds.

General:
-All of the ID's for legion and NCR are completely invisible. 
-AMR rounds are invisible.
-Fucktons of completely wide open blank spaces that could be filled with additional loot buildings, or anything at all really, its just annoying mass stretches of wasted potential.

**Comment by GoodG1448** [comment]
What about making use of that empty space just south of the NCR base to make a small solar array and put a SMES somewhere inside the NCR base?

**Comment by Bodynasr** [comment]
Buff the Legion, **they barely won a single round** since the new map kicked in. Everyone notices it and people stopped playing the legion due to it.
![legion](https://user-images.githubusercontent.com/51898651/59590748-09593180-90ed-11e9-96ed-2dd1a42a245a.PNG)


Armory: 
-Armory is hilariously empty when compared to NCR's armory
-Add more shit tier weapons such as Ithacas and scoped hunting rifles. WHY WERE BOLAS REMOVED!?
-MORE AMMUNITION, for a faction where almost all use revolvers, the revolver ammo runs dry by min 30 mark.
-Add a limited amount of Molotovs? NCR got flashbangs as well
-Bring back C4, we are tired of having to go into a frontal assault against the meta of Bonfires.
TLDR: **RETURN** IT LIKE IT USED TO BE. Every NCR recruit takes a service rifle, extra clips and a side arm with its clips as well.


General problems of the legion:

-The bonfire meta, Both NCR and legion puts bonfire everywhere and for a close range/melee faction, it simply kills the legion. I suggest deleting it, or make the fire wear out much quicker. It simply breaks immersion, take a look at new vegas, do you see any of the factions fortifications have bonfires in it? 

-Advanced Signaling devices, we used to have three of them but now we just have one? Why? Some dude runs off with one and now all slave collars have become useless.


It was a long rant but after 12 rounds of having our asses whooped hard, something needs to be done.
